Fix wrong testgrid distribution referred in the jenkins pipeline

azinneera opened the pull request:
## Purpose
> The Jenkins build fails due to the changes pattern in name of testgrid distribution.
> Resolves #449

## Goals
> Use a standard name for the distribution so that modification of the distribution name in the Jenkins pipeline is avoided

## Approach
> Renamed the file when copying in CD pipeline script
> Avoided packing the directory with milestone and version into the distribution zip

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

sameerawickramasekara commented:
It's better if we update TestGrid jenkinsfile as well with these changes

kasunbg commented on distribution/assembly.xml:
```diff
@@ -7,6 +7,7 @@
     <formats>
         <format>zip</format>
     </formats>
+    <includeBaseDirectory>false</includeBaseDirectory>
```
IMO, including the base directory is quite important. Otherwise, unpacking will lead to cumbersome directory structure..

azinneera commented:
Added changes in jenkins-pipeline in a0e8470
Revised distribution to have the base directory with version being removed in a136c69
Added a separate version file  in 4652c9d
